ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -500,8 +500,7 @@ private class JdbcLedgerDao(
         "command_id" -> (tx.commandId: Option[String]),
         "application_id" -> (tx.applicationId: Option[String]),
         "submitter" -> (tx.submittingParty: Option[String]),
-        "workflow_id" -> tx.workflowId
-          .getOrElse("unspecified workflow"), // TODO: Not allowed to insert an empty string - not a valid ledger string
+        "workflow_id" -> tx.workflowId.getOrElse(""),
```

Comment:
The workflow identifier can be empty. Not sure of how this can be represented at this level (I guess the choice is between `Option` and `null` - which you can get with `.orNull` from the `Option`).

Reply:
Unfortunately we always stored it as empty string rather than null. Hence #2446.

Reply:
I see, disregard my comment.